example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -352,12 +349,14 @@ describe('The S3 Ingest Granules workflow', () => {
 
     it('has expected payload', () => {
       expect(granule.cmrLink).toEqual(`${getUrl('search')}granules.json?concept_id=${granule.cmrConceptId}`);
+      const updatedGranule = expectedPayload.granules[0];
+      //updatedGranule.files = addUrlPathToGranuleFiles(updatedGranule.files, testId, collectionUrlString)
```

Comment:
Is it intentional to leave this commented?

Reply:
This should have been removed - the updates were pulled to the top in a refactor and this was left in.   L359 suffices in using the updatedGranule instead.